api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1924,7 +1924,7 @@ service AuthService {
   rpc GetSessionTracker(GetSessionTrackerRequest) returns (types.SessionTrackerV1);
 
   // GetActiveSessionTrackers returns a list of active sessions.
-  rpc GetActiveSessionTrackers(google.protobuf.Empty) returns (stream types.SessionTrackerV1);
+  rpc GetActiveSessionTrackers(types.SessionTrackerFilter) returns (stream types.SessionTrackerV1);
```

Comment:
I don't think this is a backward-compatible change. We could add a new method, but that would also need some caution.

Reply:
Oh yeah, derp. I'll try again with adding a new rpc.

Reply:
Try af88cfce51eb633d761b6ee4ccd7e64b2c6fbe22